scripts/eosio_build_amazon.sh

```diff
@@ -1,234 +1,336 @@
-	OS_VER=$( cat /etc/os-release | grep VERSION_ID | cut -d'=' -f2 | sed 's/[^0-9\.]//gI' | cut -d'.' -f1 )
+	OS_VER=$( grep VERSION_ID /etc/os-release | cut -d'=' -f2 | sed 's/[^0-9\.]//gI' | cut -d'.' -f1 )
 
 	MEM_MEG=$( free -m | grep Mem | tr -s ' ' | cut -d\  -f2 )
 	CPU_SPEED=$( lscpu | grep "MHz" | tr -s ' ' | cut -d\  -f3 | cut -d'.' -f1 )
 	CPU_CORE=$( lscpu | grep "^CPU(s)" | tr -s ' ' | cut -d\  -f2 )
-	MEM_GIG=$(( (($MEM_MEG / 1000) / 2) ))
-	JOBS=$(( ${MEM_GIG} > ${CPU_CORE} ? ${CPU_CORE} : ${MEM_GIG} ))
+	MEM_GIG=$(( ((MEM_MEG / 1000) / 2) ))
+	export JOBS=$(( MEM_GIG > CPU_CORE ? CPU_CORE : MEM_GIG ))
 
-	DISK_TOTAL=`df -h . | grep /dev | tr -s ' ' | cut -d\  -f2 | sed 's/[^0-9]//'`
-	DISK_AVAIL=`df -h . | grep /dev | tr -s ' ' | cut -d\  -f4 | sed 's/[^0-9]//'`
+	DISK_TOTAL=$( df -h . | grep /dev | tr -s ' ' | cut -d\  -f2 | sed 's/[^0-9]//' )
+	DISK_AVAIL=$( df -h . | grep /dev | tr -s ' ' | cut -d\  -f4 | sed 's/[^0-9]//' )
 
-	printf "\n\tOS name: $OS_NAME\n"
-	printf "\tOS Version: ${OS_VER}\n"
-	printf "\tCPU speed: ${CPU_SPEED}Mhz\n"
-	printf "\tCPU cores: $CPU_CORE\n"
-	printf "\tPhysical Memory: $MEM_MEG Mgb\n"
-	printf "\tDisk space total: ${DISK_TOTAL}G\n"
-	printf "\tDisk space available: ${DISK_AVAIL}G\n"
+	printf "\\n\\tOS name: %s\\n" "${OS_NAME}"
+	printf "\\tOS Version: %s\\n" "${OS_VER}"
+	printf "\\tCPU speed: %sMhz\\n" "${CPU_SPEED}"
+	printf "\\tCPU cores: %s\\n" "$CPU_CORE"
+	printf "\\tPhysical Memory: %s Mgb\\n" "${MEM_MEG}"
+	printf "\\tDisk space total: %sGb\\n" "${DISK_TOTAL}"
+	printf "\\tDisk space available: %sG\\n" "${DISK_AVAIL}"
 
-	if [ $MEM_MEG -lt 7000 ]; then
-		printf "\tYour system must have 7 or more Gigabytes of physical memory installed.\n"
-		printf "\texiting now.\n"
+	if [ "$MEM_MEG" -lt 7000 ]; then
```

Comment:
Can you please update [Docker requirement](https://github.com/EOSIO/eos/blob/master/Docker/README.md#docker-requirement) to 7GB?

Reply:
Done